src/training_pipeline.py

```diff
@@ -64,10 +65,18 @@ def train(config: DictConfig) -> Optional[float]:
                 logger.append(hydra.utils.instantiate(lg_conf))
 
     # Init lightning trainer
+    if config.trainer.get("amp_backend"):
+        log.info(f"Using Mixed Precision <{config.trainer.amp_backend}>")
+    if config.trainer.get("precision"):
+        log.info(f"Using precision training <{config.trainer.precision}>")
```

Comment:
A similar info is already printed by lightning at the beginning of the training
![image](https://user-images.githubusercontent.com/43458894/156948743-0401dd17-353a-4626-9c67-f9ec35f62b36.png)